hat/menupermissions/models.py

```diff
@@ -113,4 +114,5 @@ class Meta:
             ("iaso_entity_duplicates_write", ENTITY_DUPLICATES_WRITE),
             ("iaso_datastore_read", DATASTORE_READ),
             ("iaso_datastore_write", DATASTORE_WRITE),
+            ("iaso_update_org_unit_types", UPDATE_ORG_UNIT_TYPES),
```

Comment:
@olethanh is right, we probably should have a standard way of naming read/write permissions